clients/gtest/csrlsvchol_gtest.cpp

```diff
@@ -66,9 +66,15 @@ template <bool HOST>
 class CSRLSVCHOL_BASE : public ::TestWithParam<csrlsvchol_tuple>
 {
 protected:
-    CSRLSVCHOL_BASE() {}
-    virtual void SetUp() {}
-    virtual void TearDown() {}
+    void SetUp() override
+    {
+        if(hipsolverSpCreate(nullptr) == HIPSOLVER_STATUS_NOT_SUPPORTED)
+            GTEST_SKIP() << "Sparse functionality is not enabled";
```

Comment:
I think this error message should explicitly say that hipSOLVER hasn't been compiled with sparse support, just saying "enabled" might make users think there's a switch or environment variable they need to set to enable it.

Reply:
Reading further on it looks like this is as specific as we can be right here, but I still think more specific error messages would be a good idea since there are two distinct failure states for sparse not compiled in vs sparse couldn't load.